src/app/pages/rpc-sign-psbt/use-rpc-sign-psbt.tsx

```diff
@@ -54,6 +55,8 @@ export function useRpcSignPsbt() {
 
     await broadcastTx({
       tx,
+      // skip utxos check for psbt txs
+      skipBypassUtxoIdsCheckFor: decodeBitcoinTx(tx).inputs.map(input => bytesToHex(input.txid)),
```

Comment:
This naming isn't real clear to me, just bc `skip` and `bypass` are essentially the same meaning? Maybe just `skipSpendableCheckUtxoIds`?

Reply:
+1.

Don't wanna increase scope here because you've been working on this a while, but inspired by react query, `refetchOnFocus` option: one idea could be that this arg could be typed as `string[] | 'all'` to encapsulate case of bypassing everything?

Would avoid having to do this parsing to get all the txids.

Reply:
fixed naming, and changed type. thanks!